core/src/validator.rs

```diff
@@ -187,16 +192,15 @@ impl Validator {
                 let  stakes: Vec<Option<u64>> = bank
                     .epoch_vote_accounts(bank.get_epoch_and_slot_index(bank.slot()).0)
                     .iter()
-                    .filter_map(|account| match account {
-                        pubkey, (stake, accoount) => {
+                    .filter_map(|account| match (account) {
+                        Some(pubkey, (stake, account)) => {
```

Comment:
@mvines, I guess filter_map unwraps an option automatically, so  it is not necessarily to add Some here right?

Reply:
Sorry for the review delay here @cyberbono3!  I just realized that we can use [.find()](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.find) instead of `.filter_map()`.  This way `let stakes: Vec<Option<u64>> ...` turns into just `let  stake: Option<u64> ...`, we can remove the `for stake in stakes.iter()` loop entirely and just `match stake { ... }` directly.

Reply:
@mvines thanks for your feedback. I used find instead of filter_map and then matched stake directly. I tried to run cargo test from solana/core to make sure my code has no compile errors, got this [error](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=fca02dcdbe107dd79901eec811f6364c). It seems, that we use outdated librocksdb-sys v5.18.3, the current version is librocksdb-sys = "6.1.3"